client-app/pages/category.vue

```diff
@@ -101,10 +103,19 @@
             </div>
 
             <!-- View options -->
-            <ViewMode v-model:mode="savedViewMode" class="hidden md:inline-flex mr-6" />
+            <ViewMode v-model:mode="savedViewMode" class="hidden md:inline-flex mr-auto" />
+
+            <div v-if="!isMobileSidebar" class="relative ml-6 ml-auto cursor-pointer" @click="onOpenBranchesDialog">
+              <VcCheckbox :model-value="!!savedBranches.length" :disabled="loading">
+                <div
+                  v-html="$t('pages.catalog.branch_availability_filter_card.available_in', { n: savedBranches.length })"
```

Comment:
Maybe use `<i18n-t />`  instead of `v-html`
https://vue-i18n.intlify.dev/guide/advanced/component.html